tuf/client/updater.py

```diff
@@ -2334,14 +2330,15 @@ def _ensure_not_expired(self, metadata_object, metadata_rolename):
     # '1985-10-21T01:22:00Z'.)  Convert it to a unix timestamp and compare it
     # against the current time.time() (also in Unix/POSIX time format, although
     # with microseconds attached.)
-    current_time = int(time.time())
+
+    time = self.update_handler.earliest_valid_expiration_time(metadata_rolename)
```

Comment:
`Redefining name 'time' from outer scope (line 127)pylint(redefined-outer-name)`

Unused import `time` on line 127, but I would rename this variable to `earliest_exp_time` or some similar, more meaningful, name.